payment-failure/src/main/scala/com/gu/identity/paymentfailure/BrazeEmailService.scala

```diff
@@ -72,16 +72,16 @@ class DefaultBrazeEmailService(
   }
 }
 
-// (modulo errors) half of the emails sent will include the encrypted email token;
-// the other half will include no sign-in tokens.
-// There will be additional meta data (abName and abVariant) to facilitate tracking which segment a user is in.
-class BrazeEmailServiceWithEncryptedBrazeEmailTest(
+// Used to send an email with additional trigger properties (aka Braze metadata) derived from a variant in an AB test.
+// For example abName, abVariant and an additional token to facilitate sign-in.
+// See e.g. EncryptedEmailTest for a concrete example.
+class BrazeEmailServiceWithAbTest(
```

Comment:
This has been made slightly more generic so an email can be sent with an arbitrary AB test.